analysis/04_April24_activation_status_copernicus.qmd

```diff
@@ -0,0 +1,198 @@
+---
+title: ""
+format:
+  html:
+    toc: true
+    toc-depth: 3
+    code-fold: true
+    self-contained: true
+    embed-resoures: true
+    smooth-scroll: true
+execute:
+  include: true
+  echo: true
+  warning: false
+  message: false
+  eval: true
+  results: "asis"
+  out.width: "100%"
+  code-fold: true
+editor: visual
+project:
+  execute-dir: project
+---
+
+## Intro
+
+As we are quickly needing to shift forecast data sources for AWS bucket to Copernicus Data Store (CDs), I've the relevant part from the email generation script and added it to this notebook just for a quick review. This will allow reviewer to quickly review the threshold comparison and activation status update without needing to review all of the pipeline code. 
+
+Once approved I willl merge this into the `nrt-monitoring` and eventually `main` so emails are generated only from CDs
+
+```{r}
+#' Description
+#' Script for monitoring CADC trigger.
+#' It needs to be run manually as natl forecast pipeline is not predictable.
+#' Unfortunately, no getting around this as we need to manually check this data everytime.
+#' 
+#' Some aspects, of script are being set for eventual migration to GH Action, but in other
+#' components this approach was abandoned -- eventually could be refactored to GH Action if
+#' data pipelines become stable.
+
+library(aws.s3)
+library(tidyverse)
+library(sf)
+library(googledrive)
+library(exactextractr)
+library(fs)
+library(gt)
+library(glue)
+library(gghdx)
+library(blastula)
+library(here)
+library(targets) # should remove for GHA eventually
+library(terra)
+gghdx()
+
+fp_email_util_funcs <- list.files(
+  file.path("src","email","email_utils"),
```

Comment:
I just had to change this to `file.path("..", "src", "email", "email_utils")` but I presume you were going to refactor stuff anyways

Reply:
[48bf0ee](https://github.com/OCHA-DAP/ds-aa-lac-dry-corridor/pull/16/commits/48bf0ee6d8e5f5a763a7fd82b7ff0e2bc9f670ca)